CMakeLists.txt

```diff
@@ -1,31 +1,28 @@
-cmake_minimum_required(VERSION 3.18)
+cmake_minimum_required(VERSION 3.15...3.18)
 
 project(scikit_build_example VERSION "0.0.1")
 
-# Define CMAKE_INSTALL_xxx: LIBDIR, INCLUDEDIR
-include(GNUInstallDirs)
```

Comment:
This actually breaks conda, since conda defines these (and the .so then gets put in the wrong place for an extension).